client/src/components/body/EducatorResources/ResourcesSection.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+import {
+  GROUP_OVERVIEW,
+  LAB_OVERVIEW,
+} from "../../../constants/educatorResources/LabOverview";
+import LabService from "../../../services/LabService";
+import ResourceRow from "./components/ResourceRow";
+
+const ResourcesSection = () => {
+  const [displayedResource, setDisplayedResource] = useState("");
+  const [labs, setLabs] = useState([]);
+  const getLabs = async () => {
+    const labData = await LabService.getAllLabs();
+    setLabs(labData);
+  };
+
+  useEffect(() => {
+    setDisplayedResource("Overview");
```

Comment:
nit: should we just call `displayOverview` here?

Reply:
or we can set the default state to `Overview`

Reply:
Yep, set the initialState to overview